meta-schema.graphql

```diff
@@ -0,0 +1,115 @@
+# Base types
+type Party {
+	id:ID!
+	idType:String! #PartyId, IPI, ISNI, MBID
+	displayName:String!
+	firstName: String
+	lastName: String
+}
+
+type Organisation {
+	id:String!
+	idType:String! #DPid, IPI, MBID
+	displayName:String!
+	formalName:String
+}
+
+# Music Types
+type TerritoryAgreement {
+	territoryCode:Int! #CISAC Territory standard
+	condition:String! # CWR inclusion/exclusion indicator
+}
+
+type InterestedPartyAgreement {
+	agreementRoleCode:String!
+	interestedPartyNumber:Int!
+	ipi:Int
+	ipiBase:Int
+	lastName:String!
+	firstName:String
+	prSociety:String
+	prShare:Int
+	mrSociety:String
+	mrShare:Int
+}
+
+type MusicWork {
+	iswc:String
+	workTitle:String!
+	territories:[TerritoryAgreement]
+	interestedParty:[InterestedPartyAgreement]
+	publishers:[Organisation]
+	composers:[MusicComposer]
+	recordings:[MusicRecording]
+}
+
+# DDEX:ERN:SoundRecording
+type MusicRecording {
+	isrc: String
+	iswc: String
```

Comment:
Do recordings have an ISWC? I thought that comes from the `MusicWork`?

Reply:
It does yeah, but the recording is of a certain underlying work. It is rare to be present in ERNs tbh.